test/gerkin/features/data_update_entries.feature

```diff
@@ -1,22 +1,19 @@
 Feature: API endpoint allowing users to update multiple records in the Digital Registries database.
   Request endpoint: PUT /data/{registryname}/{versionnumber}/update-entries
 
-  Background:
-    Given The user wants to update multiple records in the Digital Registries database
-
   Scenario: The user successfully updates two existing records in the Digital Registries database
-    Given The records exist in the database
-    When The user triggers an action to update two records in the database
-    And The request with a valid payload is sent
-    Then Operation finishes successfully
+    Given The user wants to update multiple records with first name "Alfie" in the Digital Registries database
+    And The records with first name "Alfie" exist in the database
+    When The user triggers an action to update records with first name "Alfie" in the database
+    Then Operation to update records with first name "Alfie" finishes successfully
 
   Scenario: The user is not able to update two records which do not exist in the Digital Registries database
-    Given The records do not exist in the database
-    When The user triggers an action to update two records in the database
-    And The request with a valid payload is sent
-    Then Operation results in an error
+    Given The user wants to update multiple records with first name "Jerry" in the Digital Registries database
+    And The records with first name "Jerry" do not exist in the database
+    When The user triggers an action to update records with first name "Jerry" in the database
+    Then Operation results to update records  with first name "Jerry" is an error
 
-  Scenario: The user is not able to update two records in the Digital Registries database
-    When The user triggers an action to update two records in the database
-    And The request with an invalid payload is sent
-    Then Operation results in an error
+  Scenario: The user is not able to update two records in the Digital Registries database because of an invalid request
+    Given The user wants to update multiple records with name "Jasmine" in the Digital Registries database
+    When The user triggers an action to update records with name "Jasmine" in the database
+    Then Operation results to update record with name "Jasmine" is an error
```

Comment:
```suggestion
     The result of an operation to update records with the first name 'Jasmine' returns an error
```
Additionally, should we add any information about invalid request to any of Given/When/Then statements. These looks very similar to previous test description.

Reply:
This information is in the title of the scenario. We can discuss it in the stand-up meeting because most scenarios are written like this :)